src/runtime/main.ts

```diff
@@ -41,14 +44,20 @@ export function revive(islands: Record<string, ComponentType>, props: any[]) {
       startNode.parentNode!.removeChild(startNode); // remove start tag node
 
       const [id, n] = tag.split(":");
-      render(
-        h(islands[id], props[Number(n)]),
-        createRootFragment(
-          parent! as HTMLElement,
-          children,
-          // deno-lint-ignore no-explicit-any
-        ) as any as HTMLElement,
-      );
+      const _render = () => {
+        render(
+          h(islands[id], props[Number(n)]),
+          createRootFragment(
+            parent! as HTMLElement,
+            children,
+            // deno-lint-ignore no-explicit-any
+          ) as any as HTMLElement,
+        );
+      };
```

Comment:
```suggestion
      function _render() {
        render(
          h(islands[id], props[Number(n)]),
          createRootFragment(
            parent! as HTMLElement,
            children,
            // deno-lint-ignore no-explicit-any
          ) as any as HTMLElement,
        );
      }
```
Nit: arrow functions are only really used with anonymous functions in this codebase.

Reply:
Could you please also resolve the merge conflict?